### ISSUE TITLE
Update to OpenTK 3.0.1

### ISSUE THREAD

**Issue by MutterOberin** [opcon/QuickFont#43]
In OpenTK 3.0.1 the signature of `GL.BlendFunc` changed. The enums are `BlendFactor` instead of `BlendFactorSrc` and `BlendFactor` instead of `BlendFactorDest`

Update the call QFontDrawing accordingly:

```
if (_useDefaultBlendFunction)
            {
                GL.Enable(EnableCap.Blend);
                GL.BlendFunc(BlendingFactor.SrcAlpha, BlendingFactor.OneMinusSrcAlpha);
            }
```

**Comment by MutterOberin** [comment]
Also I set the `UseDefaultBlendFunction` to  `false` and it still threw an "Method not found exception".

**Comment by opcon** [comment]
Fixed in #45
